bluebottle/analytics/management/commands/export_analytics_data.py

```diff
@@ -62,17 +62,23 @@ def handle(self, **options):
     @override_settings(ANALYTICS_ENABLED=True)
     def _process(self, cls, start_date, end_date):
         cls_name = cls.__name__
+        # TODO: Come up with a better way of generically getting the timestamp for influxdb log entry
```

Comment:
Yes that would be very much preferred to have a consistent way of setting created and updated fields across all models.